nginx/Dockerfile

```diff
@@ -24,5 +27,5 @@ RUN mkdir -p /tmp/nginx && \
 RUN touch /var/run/nginx.pid && \
     chown -R nginx:nginx /var/run/nginx.pid
 
-# Run master process as user nginx
-USER nginx
\ No newline at end of file
+# Run master process as non-root user
+USER 1000
```

Comment:
when using user ngnix, we get an error in radix
![image](https://user-images.githubusercontent.com/69512295/175948472-38ce9690-350c-40ef-b221-d2eff9b5372d.png)